### DISCUSSION TITLE
Deleted

### DISCUSSION THREAD

**Discussion by ghost** [pyinstaller/pyinstaller#6608] [PyInstaller]

**Comment by rokm** [comment]
You can unpack the PyInstaller executable-embedded archive and then decompile the collected .pyc modules (provided they have not been obfuscated by a 3rd party utility), if that's what you're asking. 

For viewing/extracting files from embedded archive, you can use PyInstaller's [`pyi-archive_viewer` utility](https://github.com/pyinstaller/pyinstaller/blob/develop/archive_viewer.py) or a more specialized tool, such as [pyinstxtractor](https://github.com/extremecoders-re/pyinstxtractor). Or you can roll your own utility, based on PyInstaller's archive readers/writers code.
